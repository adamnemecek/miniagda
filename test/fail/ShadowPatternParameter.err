--- opening "ShadowPatternParameter.ma" ---
--- scope checking ---
scope check error: D
/// c
/// TBind {boundDec = Dec {thePolarity = ^}, boundNames = [n], boundType = Nat}: Identifier n already in context
