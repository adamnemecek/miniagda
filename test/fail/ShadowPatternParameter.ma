-- 2013-04-06

data Nat { zero ; suc (n : Nat) }

data D (n : Nat)
{ c (n : Nat) : D (suc n)
}
