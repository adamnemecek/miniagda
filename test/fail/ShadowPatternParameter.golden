MiniAgda by Andreas Abel and Karl Mehltretter
--- opening "ShadowPatternParameter.ma" ---
--- scope checking ---
scope check error: D
/// c
/// TBind {boundDec = Dec {thePolarity = ^}, boundNames = [n], boundType = Nat}: Identifier n already in context
